frontend/model/state.js

```diff
@@ -355,6 +355,9 @@ const getters = {
   groupIncomeDistribution (state, getters) {
     return groupIncomeDistribution({ state, getters, monthstamp: currentMonthstamp() })
   },
+  groupAdjustedIncomeDistribution (state, getters) {
+    return groupIncomeDistribution({ state, getters, monthstamp: currentMonthstamp(), adjusted: true })
+  },
```

Comment:
Why is this being created instead of using the existing `groupIncomeAdjustedDistribution` which does the same thing?

Reply:
You're probably right, it's probably the same thing. I looked for this but didn't find it at first. I'll take a closer look at this code again Monday.

But the bigger question is about "ourContributionSummary" which is described in [this comment](https://github.com/okTurtles/group-income-simple/issues/763#issuecomment-704541338). That's the real thing I need feedback here, and the main reason I opened this WIP PR.